docs/tracing_ui.md

```diff
@@ -0,0 +1,210 @@
+
+2. To import a skeleton tracing as a completely new wK annotation, just drag and drop the NML file anywhere on your user dashboard. Alternately, navigate to your user dashboard and use the `Upload Annotation` button within the "Explorative Tracings" section. 
+
+![Skeletons can be imported by drag and drop in the tracing view or from the dashboard](images/tracing_ui_import.png)
+
+### Merging Skeleton Annotation
+There are two ways for merging tracings:
+
+1. While in the tracing UI, drag and drop an NML file onto your browser window to import a skeleton. The imported skeleton will be merged with currently open tracing.
+
+2. If you would like to merge your current tracing with another existing tracing, select the `Merge` operation from the overflow menu next to the `Save` button. (see image) Either enter the ID of an existing explorative tracing or select a whole project and proceed to merge the selection with your currently open tracing. The resulting annotation can either be created as a new explorative tracing. Otherwise, the merge will happen in your current tracing. 
+
+![1. Select the Merge operation from the menu](images/tracing_ui_merge_1.png)
+![2. Merging can be done with a whole wK project or using the ID of an existing explorative tracing](images/tracing_ui_merge_2.png)
+
+
+## Volume Annotations
+- what are volume tracings
+- tools
+  - brush, move, selection
+- adding / deleting segmentation
+- keyboard shortcuts
+- segementation tab
+
+
+## Tracing UI Settings
+The settings menu allows users to fine-tune some parameters of webKnossos. All settings are automatically saved as part of a user's profile. 
+The `Tracing` settings include options related to interacting with a dataset while annotating, e.g. mouse movement speed. Tracing settings only affect the currently open annotation and will be restored when reopening the respectibe annotation in the future.
+The `Dataset` settings include options to adjust the rendering of the dataset, e.g. brightness & contrast. Dataset settings effect all of the user's annotations referencing this particular dataset so that tracing can be created using the same conditions.
+
+Not all settings are available in every tracing mode.
+
+### Tracing Settings
+Controls
+- `Keyboard delay (ms)`: The initial delay before an operation will be executed when pressing a keyboard shortcut. A low value will immediately execute a keyboard's associated operation, whereas a high value will delay the execution of an operation. This is useful for preventing an operation being called multiple times when rapidly pressing a key in short succession, e.g. for movement.
+
+- `Move Value (nm/s)`:  A high value will speed up movement through the dataset, e.g. when holding down the spacebar. Vice-versa, a low value will slow down the movement allowing for more precision. This setting is especially useful in `Flight mode`.
+
+- `d/f-Switching`: ¯\_(ツ)_/¯
```

Comment:
How about: "When d/f-Switching is enabled, the direction of the shortcuts d (move backward) and f (move forward) will invert, when you create nodes while going backwards. As a result, f will always move you into the direction you are currently tracing (vice versa for d). When the option is disabled, 
the shortcuts are independent of  your current tracing direction."